src/main/java/com/example/bcsd/controller/PostController.java

```diff
@@ -0,0 +1,30 @@
+package com.example.bcsd.controller;
+
+import com.example.bcsd.dto.BoardResponseDto;
+import com.example.bcsd.dto.PostResponseDto;
+import com.example.bcsd.service.ArticleService;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@RequestMapping("/posts")
+@Controller
+public class PostController {
+
+    private final ArticleService articleService;
+
+    public PostController(ArticleService articleService) {
+        this.articleService = articleService;
+    }
+
+    @GetMapping
+    public String getPostByBoardId(@RequestParam(name = "boardId", required = false, defaultValue = "World") Long id, Model model){
```

Comment:
이 부분도 개행 해주세요! 보통 코인 프로젝트에서는 아래와 같은 방식으로 개행합니다 
```suggestion
    public String getPostByBoardId(
        @RequestParam(name = "boardId", required = false, defaultValue = "World") Long id, 
        Model model
    ){
```

Reply:
감사합니다!